src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -359,23 +359,22 @@ Obtains certificates using the ACME protocol. Note that `acme` is a default issu
 
 	- **any_common_name** <span id="any_common_name"/> is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names.
 
+##### ZeroSSL
+
+Caddy will implicitly use [ZeroSSL's ACME endpoint](https://zerossl.com/documentation/acme/) (and generate EAB credentials) if you specify the the [`email` global option](/docs/caddyfile/options#email).
+
+To provide your own EAB credentials for ZeroSSL, specify the `dir` and `eab` options.
 
 #### zerossl
 
-Obtains certificates using the ACME protocol, specifically with ZeroSSL. Note that `zerossl` is a default issuer, so configuring it explicitly is usually unnecessary.
+Obtains certificates using the ZeroSSL API.
 
 ```caddy-d
-... zerossl [<api_key>] {
+... zerossl <api_key> {
 	...
```

Comment:
The omitted fields here should be documented.

Reply:
Good point, here they are from the Godoc:

```go
//	... zerossl <api_key> {
//		    validity_days <days>
//		    alt_http_port <port>
//		    dns <provider_name> ...
//		    propagation_delay <duration>
//		    propagation_timeout <duration>
//		    resolvers <list...>
//		    dns_ttl <duration>
//	}
```

Reply:
Added, save for describing `validity_days`.

Reply:
Thanks. Maybe we should refer the users to the [ZeroSSL API docs](https://zerossl.com/documentation/api/create-certificate/) anyway. But for that field it's basically either 365 or 90.